src/runtime-rs/crates/hypervisor/src/ch/inner_hypervisor.rs

```diff
@@ -741,23 +741,29 @@ impl CloudHypervisorInner {
     }
 
     pub(crate) async fn get_hypervisor_metrics(&self) -> Result<String> {
-        todo!()
+        // No metrics currently - see https://github.com/kata-containers/kata-containers/issues/8800
+
+        Ok(String::new())
     }
 
-    pub(crate) fn set_capabilities(&mut self, _flag: CapabilityBits) {
-        todo!()
+    pub(crate) fn set_capabilities(&mut self, flag: CapabilityBits) {
+        let mut caps = Capabilities::default();
+
+        caps.set(flag)
     }
 
-    pub(crate) fn set_guest_memory_block_size(&mut self, _size: u32) {
-        todo!()
+    pub(crate) fn set_guest_memory_block_size(&mut self, size: u32) {
+        self._guest_memory_block_size_mb = size;
     }
 
     pub(crate) fn guest_memory_block_size_mb(&self) -> u32 {
-        todo!()
+        self._guest_memory_block_size_mb
     }
 
     pub(crate) fn resize_memory(&self, _new_mem_mb: u32) -> Result<(u32, MemoryConfig)> {
-        todo!()
+        // No resize currently - see https://github.com/kata-containers/kata-containers/issues/8801
```

Comment:
@jodh-intel, I think it's worth adding a message here saying that the resize is not currently implemented.